ios/Plugin/InAppBrowserPlugin.swift

```diff
@@ -174,7 +193,7 @@ public class InAppBrowserPlugin: CAPPlugin {
             self.navigationWebViewController?.navigationBar.isTranslucent = false
             self.navigationWebViewController?.toolbar.isTranslucent = false
             self.navigationWebViewController?.navigationBar.backgroundColor = .white
-            self.navigationWebViewController?.toolbar.backgroundColor = .white
+            self.navigationWebViewController?.toolbar.backgroundColor = UIColor(hexString: call.getString("toolbarColor","#ffffff"))
```

Comment:
There is no documentation for this `toolbarColor` in the readme.
If I understand correctly this `toolbarColor` will get passed from JS -> Java and the input for that is documented [here](https://github.com/Cap-go/capacitor-inappbrowser#openwebviewoptions)